index.html

```diff
@@ -5,6 +5,37 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
   </head>
-  <body></body>
+  <body>
+    <nav class="navbar">
+      <div class="navbar__logo">Ahmed</div>
+      <div class="navbar__hamburger">
+        <img src="images/Union.svg" alt="menu" />
+      </div>
+    </nav>
+
+    <header class="header">
+      <h1 class="header__title">I’m Ahmed Glad to see you!</h1>
+      <p class="header__text">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <a href="#" class="header__link">Let's connect</a>
+
+      <div class="header__icons">
+        <img src="images/linkedin.svg" alt="linkedin" />
+        <img src="images/twitter.svg" alt="twitter" />
+        <img src="images/medium.svg" alt="medium" />
+        <img src="images/github.svg" alt="github" />
+      </div>
```

Comment:
I can see you did a great job aligning the icons. But think they are presented in a list and it is great that we already have html tags that handle lists. How about we use those?

- [ ] The list of icons uses a `<ul>` tag to group them, and `<li>` per icon; each icon has a link (a tag).